### PR TITLE
Optimize db, setup CRUD, and k6 stress test

### PR DESCRIPTION
**Pull request by kelsyvghn**

### CODE REVIEW

File: db/postgres/schema.sql [hunk 1]
```diff
@@ -14,42 +14,50 @@ CREATE TABLE tracks (
...
-ALTER TABLE tracks ADD FOREIGN KEY (genre) REFERENCES genres (id);
-ALTER TABLE playlists ADD FOREIGN KEY (genre) REFERENCES genres (id);
+-- ALTER TABLE tracks ADD FOREIGN KEY (genre) REFERENCES genres (id);
+-- ALTER TABLE playlists ADD FOREIGN KEY (genre) REFERENCES genres (id);
 ALTER TABLE playlists ADD FOREIGN KEY (tracks) REFERENCES tracks (id);
-ALTER TABLE genres ADD FOREIGN KEY (songs) REFERENCES tracks (id);
\ No newline at end of file
+-- ALTER TABLE genres ADD FOREIGN KEY (songs) REFERENCES tracks (id);
+
+SELECT * from playlists INNER JOIN genres on tracks.genre FROM tracks.genre = genres.id where id = 6; CREATE INDEX artist ON tracks USING hash (column);
```
Comment (phuctranlv) [inline db/postgres/schema.sql:61]: this looks crazy 😮

File: tests/stressTest2.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export let options = {
+  vus: 600,
+  duration: '30s'
+};
+
+export default function() {
```
Comment (phuctranlv) [inline tests/stressTest2.js:9]: interesting approach

**Review by phuctranlv — APPROVED** [review]
nice job!

File: db/postgres/schema.sql (hunk 1 above)
Comment (kelsyvghn) [reply db/postgres/schema.sql:61]: haha, fortunately I didn't need it.

File: tests/stressTest2.js (hunk 2 above)
Comment (kelsyvghn) [reply tests/stressTest2.js:9]: this one didn't really work, I think I'll toy with it, but the other worked well anyhow.

**Pull request merged**